Update ip-ranges.mdx (#236)
hashicorp/terraform-docs-common: website/docs/cloud-docs/architectural-details/ip-ranges.mdx

Terraform Cloud uses static IP ranges for certain features such as notifications and VCS connections. These IP ranges are retrievable through the [IP Ranges API](/cloud-docs/api-docs/ip-ranges).

> -> **Note:** The IP ranges for each feature returned by the IP Ranges API may overlap. Additionally, these published ranges do not currently allow for execution of Terraform runs against local resources, for e.g. CLI-driven runs.

Reviewer: ```suggestion
-> **Note:** The IP ranges for each feature returned by the IP Ranges API may overlap. Additionally, these published ranges do not currently allow for execution of Terraform runs against local resources, such as for CLI-driven runs.
```

Suggest changing "for e.g." to "such as for" to avoid Latin terms and abbreviations

> Since Terraform Cloud is a shared service, the use of these IP ranges to permit access to restricted resources and their impact on your security posture should be carefully considered. Additionally, these IP ranges may change. While changes are not expected to be frequent, we strongly recommend checking the IP Ranges API every 24 hours for the most up-to-date information if you do choose to make use of these ranges.

Reviewer: ```suggestion
Since Terraform Cloud is a shared service, the use of these IP ranges to permit access to restricted resources and you should carefully consider their impact on your security posture. Additionally, these IP ranges may change. While changes are unlikely to be frequent, we strongly recommend checking the IP Ranges API every 24 hours for the most up-to-date information if you do choose to make use of these ranges.

Suggest editing to avoid passive voice style issues.
```
